counterpartylib/lib/api.py

```diff
@@ -382,21 +384,34 @@ def run(self):
 
 class APIServer(threading.Thread):
     """Handle JSON-RPC API calls."""
-    def __init__(self):
+    def __init__(self, db=None):
+        self.db = db
```

Comment:
This seems to be from https://github.com/rubensayshi/counterparty-lib/pull/10/files